ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -257,6 +309,14 @@ export default class TestDataModal extends React.Component {
     this.closeModal();
   };
 
+  excise = (subString, fromString) =>
```

Comment:
Can we use another word for `excise`? It's kinda hard to understand. Maybe too elevated?